locales/en.json

```diff
@@ -26,4 +26,7 @@
     "youremail@example.com": "youremail@example.com",
     "1 post": "1 post",
     "% posts": "% posts"
+    "< 1 min read": "< 1 min read",
+    "1 min read": "1 min read",
+    "% min read": "% min read",
 }
```

Comment:
Thanks, I will merge this but please first correct the following:
- Some days ago, they removed the `"< 1 min read"` option, so this line should be removed (see Ghost issue [#9573](https://github.com/TryGhost/Ghost/pull/9573)).
- There is a missing comma at the end of the line `"% posts": "% posts"`.
- There is an unnecessary comma at the end of the final line.

That is, before merging, this pull request should be changed to:
```
    "% posts": "% posts",
    "1 min read": "1 min read",
    "% min read": "% min read"
}
```
Thanks again.

Reply:
Sure! Thanks ;)